packages/main/src/FileUploader.ts

```diff
@@ -303,7 +345,15 @@ class FileUploader extends UI5Element implements IFormInputElement {
 	}
 
 	_onChange(e: Event) {
-		const changedFiles = (e.target as HTMLInputElement).files;
+		let changedFiles = (e.target as HTMLInputElement).files;
+
+		if (changedFiles) {
+			changedFiles = this._validateFiles(changedFiles);
+		}
+
+		if (!this.value && !changedFiles?.length) {
```

Comment:
Should the `&&` rather be `||` ?

Reply:
No, we dont want the change event firing when there's nothing previously and users put invalid files (since nothing has changed) aka when theres no value and the changed files is an empty filelist.